src/encryption.test.ts

```diff
@@ -154,7 +154,7 @@ describe('encryption', function () {
       it('should throw if passed null public key', function () {
         expect(() =>
           encrypt({
-            publicKey: null,
+            publicKey: null as any,
```

Comment:
Nit: We could use `// @ts-expect-error` in each of these places rather than casting to `any`. It's a bit safer in that we get notified if the type error goes away.

It also prevents the `any` type "leaking"  onto other lines where the type checking would still be valuable (not applicable here because `null` is a literal, but it might apply if this was done to a variable).